compiler/test-resources/scripting/classpathReport_scalacli.sc

```diff
@@ -1,8 +1,8 @@
-#!bin/scala -classpath 'dist/target/pack/lib/*'
-
+#!/usr/bin/env bin/scala
+// This file is a Scala CLI script.
 import java.nio.file.Paths
 
-def main(args: Array[String]): Unit =
+// def main(args: Array[String]): Unit = // MIGRATION: Scala CLI expects `*.sc` files to be straight-line code
```

Comment:
I wonder if it makes sense to keep this comment, or can we just remove it.
it's stated earlier that this is a Scala CLI script.
```suggestion
```